packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -1277,6 +1277,808 @@ curl -v -X DELETE \
+                }
+            },
+            "self": {
+                "href": "http://rain.okta1.com:1802/api/v1/roles/SUPER_ADMIN/subscriptions/USER_DEPROVISION"
+            }
+        }
+    },
+    {
+        "notificationType": "REPORT_SUSPICIOUS_ACTIVITY",
+        "channels": [
+            "email"
+        ],
+        "status": "subscribed",
+        "_links": {
+            "unsubscribe": {
+                "href": "http://rain.okta1.com:1802/api/v1/roles/SUPER_ADMIN/subscriptions/REPORT_SUSPICIOUS_ACTIVITY/unsubscribe",
+                "hints": {
+                    "allow": [
+                        "POST"
+                    ]
+                }
+            },
+            "self": {
+                "href": "http://rain.okta1.com:1802/api/v1/roles/SUPER_ADMIN/subscriptions/REPORT_SUSPICIOUS_ACTIVITY"
+            }
+        }
+    }
+]
+```
+
+#### Get subscription of a Role with a specific type
+
+<ApiLifecycle access="ea" />
+
+<ApiOperation method="get" url="/api/v1/roles/${roleType}/subscriptions/${notificationType}" />
+
+Get subscriptions of a Role with a specific type
+
+##### Request parameters
+
+
+| Parameter        | Description                                            | Param Type  | DataType                                    | Required  |
+| :--------------- | :----------------------------------------------------- | :---------- | :------------------------------------------ | :-------- |
+| roleType         | `type` of a Role                                       | URL         | [Role Type](#role-types)                    | TRUE      |
+| notificationType | `type` of a notification                               | URL         | [Notification Type](#notification-type)     | TRUE      |
+
+##### Response parameters
+
+
+[Subscription](#subscription-object)
+
+##### Request example
+
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/roles/SUPER_ADMIN/subscriptions/IWA_AGENT"
+```
+
+##### Response example
+
+
+```json
+{
+    "notificationType": "IWA_AGENT",
+    "channels": [
+        "email"
+    ],
+    "status": "subscribed",
+    "_links": {
+        "unsubscribe": {
+            "href": "http://rain.okta1.com:1802/api/v1/roles/SUPER_ADMIN/subscriptions/IWA_AGENT/unsubscribe",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        },
+        "self": {
+            "href": "http://rain.okta1.com:1802/api/v1/roles/SUPER_ADMIN/subscriptions/IWA_AGENT"
+        }
+    }
+}
+```
+
+#### Subscribe to a specific type
+
+<ApiLifecycle access="ea" />
+
+<ApiOperation method="post" url="/api/v1/roles/${roleType}/subscriptions/${notificationType}/subscribe" />
+
+Make a Role subscribes to a specific type. When you change the subscription status of a role, it will override the subscription of any individual user of that role.
```

Comment:
```suggestion
Subscribes a Role to a specific type. When you change the subscription status of a role, it will override the subscription of any individual user of that role.
```